pkg/apis/devops/v1alpha1/s2irun_types.go

```diff
@@ -61,8 +61,50 @@ type S2iRunStatus struct {
 	LogURL string `json:"logURL,omitempty"`
 	//KubernetesJobName is the job name in k8s
 	KubernetesJobName string `json:"kubernetesJobName,omitempty"`
+
+	// S2i build result info.
+	S2iBuildResult *S2iBuildResult `json:"s2iBuildResult,omitempty"`
+	// S2i build source info.
+	S2iBuildSource *S2iBuildSource `json:"s2iBuildSource,omitempty"`
+}
+
+type S2iBuildResult struct {
 	//ImageName is the name of artifact
 	ImageName string `json:"imageName,omitempty"`
+	//The size in bytes of the image
+	ImageSize int64 `json:"imageSize,omitempty"`
+	// Image ID.
+	ImageID string `json:"imageID,omitempty"`
+	// Image created time.
+	ImageCreated string `json:"imageCreated,omitempty"`
```

Comment:
The type of the field should not be string. Usually metav1.Time is used to represent time in k8s.

 At the same time, the field name is not direct enough. It can be named CreationTime/CreationTimpestamp.

Reply:
this field come from docker inspect api